cmake/sample_defs/cpu1_platform_cfg.h

```diff
@@ -18,23 +18,24 @@
 **  limitations under the License.
 */
 
-/******************************************************************************
-** File: cfe_platform_cfg.h
-**
-** Purpose:
-**   This header file contains the platform configuration parameters.
-**
-** Notes:
-**   The impact of changing these configurations from their default value is
-**   not yet documented.  Changing these values may impact the performance
-**   and functionality of the system.
-**
-** Author:   R.McGraw/SSI
-**
-******************************************************************************/
-
-#ifndef _cfe_platform_cfg_
-#define _cfe_platform_cfg_
+/**
+ * @file
+ *
+ *
```

Comment:
Extra line?  Looks like it's in most (all?) of the replaced file documentation

Reply:
maybe replace all `/n */n */n` with `/n */n' and (nitpick) then all '/n */n *///n` with '/n *///n` (remove extra empty comment line at the end f I got the escapes right)

Reply:
Yeah I noticed that too - the script I used didn't quite get it all right.  I will run another pass to get multiple blank lines in comments.  actually a simple "uniq" might do it.

Reply:
Addressed in commit 0780d12